src/scss/components/_progress-indicator.scss

```diff
@@ -18,6 +18,14 @@
     @include body--big;
     @include margin(-$grid-container-margin, $properties: margin-left);
 
+    // style for replacing div with a button for accessibility
+    background: none;
+    color: inherit;
```

Comment:
the `@include body` sets the color already to `var(--of-color-fg)` which is correct.

Reply:
I will double-chek but in this case the Button kept on forcing its default button background.

Reply:
could be a FF vs. Brave quirk :)